utils/commons/build_data.go

```diff
@@ -0,0 +1,19 @@
+package commons
+
+import "fmt"
+
+var (
+	appName = "SSV-Node"
+	version = "latest"
+)
+
+// SetBuildData updates local vars for build data
+func SetBuildData(app string, ver string) {
+	appName = app
+	version = ver
+}
+
+// GetBuildData returns build data
+func GetBuildData() string {
+	return fmt.Sprintf("%s:%s", appName, version)
```

Comment:
@amir-blox didn't we want to add operator pub key somehow?

Reply:
I remember we decided to avoid it for now and wait for a proper way of authenticating peers, but yes we could easily add this now until we are ready